dataflow/src/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -876,9 +879,56 @@ public static ControlFlowGraph process(ControlFlowGraph cfg) {
                 }
             }
 
+            // remove generated trees when corresponding nodes are unreachable
```

Comment:
Can you add a few more explanations of the steps below? I don't see where anything gets removed, you're building up the map of generatedTrees.